### PR TITLE
CSS functions that are not value functions.

### PR DESCRIPTION
**Pull request by estelle**
addresses the concerns from https://github.com/mdn/content/issues/20611

### CODE REVIEW

**Comment by dipikabh** [comment]
Taking a look at this shortly

File: files/en-us/web/css/css_functions/index.md [hunk 1]
```diff
@@ -20,6 +20,8 @@ The value syntax starts with the **name of the function**, followed by a left pa
 
 Functions can take multiple arguments, which are formatted similarly to CSS property values. Whitespace is allowed, but they are optional inside the parentheses. In some functional notations multiple arguments are separated by commas, while others use spaces.
 
+> **Note:** These CSS value functions are used as property values. The four [functional pseudo-classes](/en-US/docs/Web/CSS/Pseudo-classes#functional_pseudo-classes) are not value functions, so are not included. [Linguistic pseudo-classes](/docs/Web/CSS/Pseudo-classes#linguistic_pseudo-classes) and several [tree-structural pseudo-classes](/en-US/docs/Web/CSS/Pseudo-classes#tree-structural_pseudo-classes) also require parameter values, but are not value functions. Parentheses in [conditional at-rules](/en-US/docs/Web/CSS/At-rule#conditional_group_rules) are groupings, not functions.
```
Comment (dipikabh) [inline files/en-us/web/css/css_functions/index.md:23]: - Can we group the pseudo-classes as suggested below? 
- Note the extra trailing space at the end

```suggestion
> **Note:** The CSS value functions are used as property values and should not be confused with pseudo-classes. The [functional pseudo-classes](/en-US/docs/Web/CSS/Pseudo-classes#functional_pseudo-classes), [linguistic pseudo-classes](/docs/Web/CSS/Pseudo-classes#linguistic_pseudo-classes), and several [tree-structural pseudo-classes](/en-US/docs/Web/CSS/Pseudo-classes#tree-structural_pseudo-classes) require parameter values, but they're not value functions. The [conditional at-rules](/en-US/docs/Web/CSS/At-rule#conditional_group_rules) are also not value functions; the parentheses are used for groupings.
```

**Review by dipikabh — CHANGES_REQUESTED** [review]
Added a suggestion

File: files/en-us/web/css/css_functions/index.md [hunk 2]
```diff
@@ -20,6 +20,8 @@ The value syntax starts with the **name of the function**, followed by a left pa
 
 Functions can take multiple arguments, which are formatted similarly to CSS property values. Whitespace is allowed, but they are optional inside the parentheses. In some functional notations multiple arguments are separated by commas, while others use spaces.
 
+> **Note:** The CSS value functions are used as property values and should not be confused with pseudo-classes. The [functional pseudo-classes](/en-US/docs/Web/CSS/Pseudo-classes#functional_pseudo-classes), [linguistic pseudo-classes](/docs/Web/CSS/Pseudo-classes#linguistic_pseudo-classes), and several [tree-structural pseudo-classes](/en-US/docs/Web/CSS/Pseudo-classes#tree-structural_pseudo-classes) require parameter values, but they're not value functions. The [conditional at-rules](/en-US/docs/Web/CSS/At-rule#conditional_group_rules) are also not value functions; the parentheses are used for groupings.
```
Comment (teoli2003) [inline files/en-us/web/css/css_functions/index.md:23]:
```suggestion
> **Note:** The CSS value functions are used as property values and should not be confused with pseudo-classes. The [functional pseudo-classes](/en-US/docs/Web/CSS/Pseudo-classes#functional_pseudo-classes), [linguistic pseudo-classes](/en-US/docs/Web/CSS/Pseudo-classes#linguistic_pseudo-classes), and several [tree-structural pseudo-classes](/en-US/docs/Web/CSS/Pseudo-classes#tree-structural_pseudo-classes) require parameter values, but they're not value functions. The [conditional at-rules](/en-US/docs/Web/CSS/At-rule#conditional_group_rules) are also not value functions; the parentheses are used for groupings.
```

**Review by dipikabh — APPROVED** [review]
Looks good, thanks 👍

**Pull request merged**